src/emulator/mem/include/mem/mem.h

```diff
@@ -37,6 +37,7 @@ struct MemState {
     Allocated allocated_pages;
     std::mutex generation_mutex;
     GenerationNames generation_names;
+    std::map<Address, uint32_t> breakpoints;
```

Comment:
It feels wrong to put breakpoints in here. It doesn't belong there. Perhaps put it in host or kernel ? Or put it in GDB struct, and do memory modification there.

Reply:
I think I put it in there so I could access it from a CPUState object, but the CPU doesn't even care anymore. I think I'll put it in the Kernel struct.